tests/phpunit/tests/comment.php

```diff
@@ -271,6 +271,41 @@ public function test_get_approved_comments_with_post_id_0_should_return_empty_ar
 		$this->assertSame( array(), $found );
 	}
 
+	/**
+	 * @ticket 53962
+	 */
+	public function test_do_not_output_author_of_unapproved_comment() {
+		$_GET['replytocom'] = $this->create_unapproved_comment();
+		$expected           = 'Leave a Reply';
+		comment_form_title( false, false, false );
+
+		$this->expectOutputString( $expected );
+	}
+
+	public function test_cannot_reply_to_unapproved_comment() {
```

Comment:
```suggestion
	/**
	 * @ticket 53962
	 */
	public function test_cannot_reply_to_unapproved_comment() {
```

The ticket number docblock needs to be added for each test so `phpunit --group <ticket number>` will run all the tests against the ticket.

Reply:
Thanks @peterwilsoncc - I've made this change and the additional tests also have the docblock added.